src/RepoTokenList.sol

```diff
@@ -243,7 +243,7 @@ library RepoTokenList {
         address prev = current;
         while (current != NULL_NODE) {
             address next;
-            if (getRepoTokenMaturity(current) < block.timestamp) {
+            if (getRepoTokenMaturity(current) <= block.timestamp) {

```

Comment:
This seems to reverse a previous change. Did you mean to remove this?

Reply:
Are you referring to [this change](https://github.com/term-finance/yearn-v3-term-vault/commit/b07fc4695b1a0dba7c3b99021bc3581d32ac8954)? That's a different check, the previous change was in the `insertSorted` function.